README.md

```diff
@@ -1,2 +1,124 @@
+You should be able to use either version 1 (>v1.17) or version 2 of the AWS command line tools, but these instructions have been primarily tested with v1.18.
+To install, you can follow [amazon's instructions for v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) or install via your favorite package manager (`pip` and `conda` will at this time install v1, `homebrew` will install v2):
+- `pip install awscli`
+- `conda install -c conda-forge awscli`
+- `brew install awscli`
+
+#### Configuring your AWS credentials
+
+When you set up your account, you may have gotten a paired `AWS Access Key ID` and `AWS Secret Access Key`.
+If you did not, you can login to the [aws console](https://console.aws.amazon.com/), select "My Security Credentials" from the menu that appears clicking on your username in the upper right.
+Scroll down to "Access keys for CLI, SDK, & API access" and click the "Create Access Key" button.
+You will see a popup with your Access key ID and a link to show the Secret access key.
+
+In the terminal, run the command `aws configure` and when prompted paste in the `AWS Access Key ID` and `AWS Secret Access Key`.
+You may want to set up your `Default region name`: most of the computing resources used here run in `us-east-1`, but setting this to a different region should not affect most commands.
+The `Default output format` can be left as `None`
+
+The `aws configure` command will create a directory at `~/.aws` with a `config` file and a `credentials` file that will be required for `nextflow` commands to work smoothly with Batch and S3.
+
+#### AWS infrastructure
+
+The infrastructure on AWS (Batch queues, AMIs, etc.) is defined via [terraform](https://www.terraform.io) using the files in the `aws` directory. More details are described in [aws/setup-log.md](aws/setup-log.md)
+
+
+
+## Running Workflows
+
+The workflows for this repository are stored in the `workflows` directory, organized by task into subdirectories.
+In general, workflows should be run from the main `workflows` directory, rather than the subdirectories.
+This will allow easy access to the shared `nextflow.config` file and keep all intermediate and log files in a single location.
+In particular, `nextflow` creates a separate work directory for every task: these will appear by default (for local tasks) in the `workflows/work` directory if the command is invoked from `workflows`.
+As the work directories can get large, it is helpful to have a single location to keep track of and purge as needed.
+
+## Running locally
+
+⚠️ Running workflows in this repository locally is not something to take lightly! It is fine for a quick test, and useful for development, but note that most of the workflows here use very large data files, which will have to be downloaded locally to run. Workflow processing may require large amounts of RAM and time, so the following example commands will rarely be used, and are mostly for illustration. ⚠️
```

Comment:
How do I download the files for a local run?

Reply:
Less significant note, I think there are three sentences on one line here.

Reply:
Or should download have already happened when I ran Nextflow? Right now, Alevin doesn't like the file: 
```
Exception : [Error reading from the FASTA/Q stream. Minimum return code for left and right read was (-2). Make sure the file is valid.]
  salmon alevin was invoked improperly.
  For usage information, try salmon alevin --help
  Exiting.
```

Reply:
It looks like it should run from the s3 bucket link? I double checked my aws config, all seems fine -- I would have expected an AWS warning/error if not being able to access the files was the problem.

Reply:
Nextflow handles all the file downloads itself.

Yeah, I got that error too... I think it means something went wrong with the download? I hadn't seen it until today, and it doesn't seem to have recurred when I deleted the work directory and reran the workflow (though it has yet to complete 🤞🏼 )